sister/word_embedders.py

```diff
@@ -39,7 +39,7 @@ def get_word2vec(lang: str = "en"):
 
     print("Loading model...")
 
-    if lang == "ja":
```

Comment:
As same as above, please don't remove if block for "ja", add a new block for "hi" instead.